lib/helpable.rb

```diff
@@ -8,4 +8,32 @@ def get_team_name_by_id
     @team_name_by_id
   end
 
+  def team_data_object_creation
```

Comment:
I like the idea behind these data_object creation living in the helpable and not in FutbolCreatable